apps/web/src/pages/ItTrainingFundPage/ItTrainingFundPage.tsx

```diff
@@ -0,0 +1,578 @@
+                  defaultMessage: "Three types of learning opportunities",
+                  id: "Zewopw",
+                  description:
+                    "Heading for section describing learning opportunities",
+                })}
+              </Heading>
+              <p>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "The fund offers 3 types of training programs to IT employees.",
+                  id: "4CpKXN",
+                  description:
+                    "First paragraph describing learning opportunities",
+                })}
+              </p>
+              <div
+                data-h2-display="base(grid)"
+                data-h2-grid-template-columns="base(1fr) p-tablet(repeat(2, minmax(0, 1fr))) l-tablet(repeat(3, minmax(0, 1fr)))"
+                data-h2-gap="base(x1)"
+              >
+                <CardBasic
+                  data-h2-overflow="base(hidden)"
+                  data-h2-padding="base(0)"
+                  data-h2-display="base(flex)"
+                  data-h2-flex-direction="base(column)"
+                >
+                  <div
+                    data-h2-display="base(block) base:children[>span](block)"
+                    data-h2-padding="base(x1)"
+                    data-h2-background-color="base(gray.darkest) base:dark(foreground.shade)"
+                    data-h2-color="base:all(white)"
+                  >
+                    <span data-h2-font-size="base(h6, 1)">
+                      {intl.formatMessage({
+                        defaultMessage: "1. Online self-paced learning",
+                        id: "mdIHDc",
+                        description: "Title for an online learning card",
+                      })}
+                    </span>
+                  </div>
+                  <div
+                    data-h2-display="base(flex)"
+                    data-h2-flex-direction="base(column)"
+                    data-h2-justify-content="base(space-between)"
+                    data-h2-flex-grow="base(1)"
+                  >
+                    <img
+                      src={image1}
+                      alt=""
+                      data-h2-display="base(none) p-tablet(block)"
+                    />
+                    <div
+                      data-h2-padding="base(x1)"
+                      data-h2-flex-grow="base(1)"
+                      data-h2-display="base(flex)"
+                      data-h2-flex-direction="base(column)"
+                      data-h2-gap="base(x1)"
+                    >
+                      <p>
+                        <strong>
+                          {intl.formatMessage({
+                            defaultMessage: "Available now",
+                            id: "L6MPML",
+                            description:
+                              "Statement that something is available now",
+                          })}
+                        </strong>
```

Comment:
Unfortunately, this only sets the font-weight to 600, and the design is set to 700.
You can remove the `<strong>` tag and set the font weight in the p tag `<p data-h2-font-weight="base(700)">`.

Or set the strong tag within the defaultMessage like you did for the list items below.

Reply:
Good catch!  I've switched out all the strong tags for font-weights.
[remove strong tags](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11821/commits/82a5dc913b2efd9e4edd6790d4d3886c7584932e)

The inline strongs do get changed to font weight 700 by our intl provider.
packages/i18n/src/components/richTextElements.tsx